src/commands/new.js

```diff
@@ -29,8 +29,8 @@ function copyTo (destination) {
 module.exports = function (projectName) {
   const currentlyInProjectFolder = isGluestickProject();
 
-  // No project name, or ran from inside an existing project install in current directory if approved
-  if (!projectName || currentlyInProjectFolder) {
+  // Ran from inside an existing project, install in current directory if approved
+  if (currentlyInProjectFolder) {
```

Comment:
I checked that if you do not pass `projectName` we actually throw an error so this condition could be simplified.